<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1449</width>
    <height>1359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Wavelength (Angstroem)</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QDoubleSpinBox" name="dSB_wavelength">
        <property name="maximum">
         <double>9.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>4.200000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cB_get_wavelength">
        <property name="text">
         <string>get from data</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="10" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" rowspan="3">
    <widget class="QGroupBox" name="gB_corrections">
     <property name="title">
      <string>Corrections</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="13" column="0" colspan="3">
       <widget class="QCheckBox" name="cB_correct_elastic_peak_position">
        <property name="text">
         <string>Correct elastic peak position to 0 meV</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QDoubleSpinBox" name="dSB_vana_back_factor">
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="4">
       <widget class="QCheckBox" name="cB_det_efficency">
        <property name="text">
         <string>Detector efficiency correction</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QDoubleSpinBox" name="dSB_vanadium_temperature">
        <property name="maximum">
         <double>999.000000000000000</double>
        </property>
        <property name="value">
         <double>295.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="10" column="0" colspan="4">
       <widget class="QCheckBox" name="cB_substract_vana_back">
        <property name="text">
         <string>Subtract instrument background from vanadium</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="16" column="0" colspan="4">
       <widget class="QCheckBox" name="cB_substract_sample_back">
        <property name="text">
         <string>Subtract instrument background from sample</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="l_vanadium_temperature">
        <property name="text">
         <string>     Vanadium temperature</string>
        </property>
       </widget>
      </item>
      <item row="14" column="0" colspan="4">
       <widget class="QCheckBox" name="cB_mask_bad_detectors">
        <property name="text">
         <string>Mask bad detectors (Vanadium counts &lt; 0)</string>
        </property>
       </widget>
      </item>
      <item row="15" column="0" colspan="4">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="l_background_factor">
        <property name="text">
         <string>     Factor</string>
        </property>
       </widget>
      </item>
      <item row="17" column="0">
       <widget class="QLabel" name="l_sample_back_factor">
        <property name="text">
         <string>     Factor</string>
        </property>
       </widget>
      </item>
      <item row="17" column="1">
       <widget class="QDoubleSpinBox" name="dSB_sample_back_factor">
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="gB_normalisation_2">
     <property name="title">
      <string>Normalisation</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QRadioButton" name="rB_norm_no">
        <property name="text">
         <string>no</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rB_norm_monitor">
        <property name="text">
         <string>monitor</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QGroupBox" name="gB_tof_options">
     <property name="title">
      <string>Tof Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="l_epp_corr">
        <property name="text">
         <string>Elastic peak channel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="cB_delete_raw">
        <property name="text">
         <string>delete raw workspaces</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="SB_epp_channel">
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QGroupBox" name="gB_binning">
     <property name="title">
      <string>Binning</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="l_qmin">
        <property name="text">
         <string>q minimum</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="dSB_qmin">
        <property name="decimals">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="l_qmax">
        <property name="text">
         <string>q maximum</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="dSB_qmax">
        <property name="decimals">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="l_qstep">
        <property name="text">
         <string>q step size</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="dSB_qstep">
        <property name="decimals">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="l_dEmin">
        <property name="text">
         <string>dE mininum</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="dSB_dEmin">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>-99.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="l_dEmax">
        <property name="text">
         <string>dE maximum</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="dSB_dEmax">
        <property name="decimals">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="l_dEstep">
        <property name="text">
         <string>dE step size</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QDoubleSpinBox" name="dSB_dEstep">
        <property name="decimals">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QPushButton" name="pB_estimate">
        <property name="text">
         <string>suggest from data</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
